lib/YaST/Bootloader/BootloaderController.pm

```diff
@@ -10,7 +10,9 @@ package YaST::Bootloader::BootloaderController;
 use strict;
```

Comment:
This class should be called now `BootLoaderSettingsController` according to new names we are using for most of the screens.